### PR TITLE
Add v0.22.x to releases.md

### PR DESCRIPTION
**Pull request by dibyom**
# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#tests) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#docs) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commits)
- [ ] Release notes block has been filled in or deleted (only if no user facing changes)

_See [the contribution guide](https://github.com/tektoncd/triggers/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

```release-note
NONE
```
/kind misc

### CODE REVIEW

File: releases.md [hunk 1]
```diff
@@ -40,6 +40,13 @@ Further documentation available:
...
+### v0.22
+
+- **Latest Release**: [v0.22.1][v0-22-1] (2023-01-13) ([docs][v0-22-1-docs], [examples][v0-22-1-examples])
+- **Initial Release**: [v0.21.0][v0-22-0] (2022-11-16)
+- **End of Life**: 2023-03-15
+- **Patch Releases**: [v0.22.1][v0-22-1] [v0.21.0][v0-22-0]
+
 ### v0.21
 
 - **Latest Release**: [v0.21.0][v0-21-0] (2022-09-06) ([docs][v0-21-0-docs], [examples][v0-21-0-examples])
```
Comment (savitaashture) [inline releases.md:52]: We need to update this and remove v0.21.0 :thinking:

Comment (dibyom) [reply releases.md:52]: 🤦

Comment (dibyom) [reply releases.md:52]: Actually I'm not sure - I think the latest release is per released version? See https://github.com/tektoncd/pipeline/blob/main/releases.md And I don't think we had a patch release for v0.21 so 0.21.0 makes sense as the latest release for v0.21

Comment (savitaashture) [reply releases.md:52]: as per the Pipeline reference 

```
v0.22
Latest Release: [v0.22.1](https://github.com/tektoncd/triggers/releases/tag/v0.22.1) (2023-01-13) ([docs](https://github.com/tektoncd/triggers/tree/v0.22.1/docs#tekton-triggers), [examples](https://github.com/tektoncd/triggers/tree/v0.22.1/examples#examples))
Initial Release: [v0.22.0](https://github.com/tektoncd/triggers/releases/tag/v0.22.0) (2022-11-16)
End of Life: 2023-03-15
Patch Releases: [v0.22.1](https://github.com/tektoncd/triggers/releases/tag/v0.22.1) [v0.21.0](https://github.com/tektoncd/triggers/releases/tag/v0.21.0)
```
Initial Release for v0.22.1 would be v0.22.0 :thinking:

Comment (dibyom) [reply releases.md:52]: Fixed! PTAL @savitaashture

**Comment by dibyom** [comment]
> Error from server (BadRequest): error when creating "/home/prow/go/src/github.com/tektoncd/triggers/docs/getting-started/pipeline.yaml": admission webhook "validation.webhook.pipeline.tekton.dev" denied the request: validation failed: invalid value: cloudEvent: spec.resources.outputs[0].event-to-sink.type

**Review by savitaashture — APPROVED** [review]

**Review by savitaashture — COMMENTED** [review]
/lgtm

**Pull request merged**